### PR TITLE
Shifty

### PR DESCRIPTION
**Pull request by anoncam**

### CODE REVIEW

**Review by anoncam — COMMENTED** [review]
@mjordandotinfo can you approvesss?

**Comment by mjordandotinfo** [comment]
Can you remove [es5-ext@0.10.62](https://socket.dev/npm/package/es5-ext/overview/0.10.62)? Don't need to catch an upstream vulnerability on a dependency when a russian troll farm eventually owns whoever made this.

**Comment by anoncam** [comment]
yeah lol. let's make a good rule of thumb be republish pkgs that show vulns after they run through a hardening pipeline.

File: BOA_script.py
Comment (anoncam) [inline BOA_script.py:1]: it really is 2.0

**Review by anoncam — COMMENTED** [review]
I dont see the erroneous dep

**Pull request merged**